external/netstandard/netstandard.depproj

```diff
@@ -35,6 +35,12 @@
     <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.2" Include="System.Data.SqlClient">
       <Version>4.3.0</Version>
     </PackageReference>
+    <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.3" Include="System.Security.Cryptography.Cng">
```

Comment:
These two entries are redundant.  I also don't see anywhere that you have a netstandard1.x project using Cng.  Please revert this file completely.  Let me know if you see any issues with a clean build after doing so and I will help you fix those.